src/Domain/Command/Bus/SimpleBus.php

```diff
@@ -24,20 +25,39 @@ public function executeCommand($command)
     {
         $handler = $this->getHandler($command);
 
-        return $handler->handle($command);
+        return $handler($command);
     }
 
     /**
      * @param mixed $command
+     *
+     * @return mixed
```

Comment:
À priori ici on peut être un poil plus précis quand même:

```suggestion
     * @param object $command (Mais cette annotation peut carrément virer, si on utilise un vrai type-hint ci-dessous)
     *
     * @return callable
```